tailwind.config.js

```diff
@@ -23,6 +24,14 @@ module.exports = {
             //     120: '30rem',
             // },
         },
+        boxShadow: {
+            'theme-s': '0 1px 2px rgb(17 24 39 / 6%), 0 1px 3px rgb(17 24 39 / 10%)',
```

Comment:
Why wouldn't you just override the base styles? 🤔
It would make sense to make the shadows consistent. Or are the default ones still used somewhere?